src/tests/unit/tests/bug-filing/github/github-bug-filing-service.test.tsx

```diff
@@ -74,6 +74,13 @@ describe('GithubBugFilingServiceTest', () => {
         expect(wrapper.getElement()).toMatchSnapshot();
     });
 
+    it('renderSettingsForm: no valid settings object', () => {
+        const Component = GitHubBugFilingService.settingsForm;
+        props.settings = null;
+        const wrapper = shallow(<Component {...props} />);
+        expect(wrapper.find(TextField).props().value).toEqual('');
```

Comment:
```suggestion
        const textValue = wrapper.find(TextField).props().value
        expect(textValue).toEqual('');
```

Reply:
It might also be better to assert that you found that element from `wrapper`.
For example:

```
    const textField = wrapper.find(TextField);
    expect(textField).toBeDefined();
```